docs/package-imports.md

```diff
@@ -88,3 +88,5 @@ You can use the CLI to interact with the registry. By default the CLI points to
 ## Package versioning
 
 By default, the AEA can only handle one version per package. That is, a project should never use both `some_author/some_package_name:0.1.0` and `some_author/some_package_name:0.2.0`.
+
+If two packages with different versions are used in the same `subprocess`, then only the code from the last loaded package will be available in `sys.modules`. This can lead to inconsistencies and exceptions at runtime.
```

Comment:
@solarw / @marcofavorito  should we say more?

Reply:
I'd say it is fine

Reply:
```suggestion
If two packages with different versions are used in the same Python process, then only the code from the last loaded package will be available in `sys.modules`. This can lead to inconsistencies and exceptions at runtime.
```

Reply:
the "last loaded package" is a bit confusing. It does not mean necessarily that the package with the latest version is loaded. The "last loaded package", with the current implementation, AFAICS is not deterministic (due to randomization of unordered data structures across Python processes)